.github/workflows/publish-pr.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish PR
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+concurrency:
+  group: publish-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  publish-docker-image:
+    if: ${{ github.event.issue.pull_request && ${{ github.event.comment.body }} == '/publish' }}
```

Comment:
Sorry, I think there's an error here, I'm trying to fix it on an other project. Did you try it somehow?

Reply:
Ah yes, it may fix it: https://github.com/okp4/template-python-project/pull/19/commits/70b34db906bc86b71eb18418f67901486c3f36b8

Reply:
but yours is working as well sorry